docs/understanding-tracking-design/managing-event-specifications/ui/index.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "Managing Event Specifications in the Console"
+sidebar_label: "Using the UI"
+sidebar_position: 1
+sidebar_custom_props:
+  offerings:
+    - enterprise
+    - cloud
+---
+
+## Crafting a New Event Specification through the Console
+
+Event specifications serve as direct counterparts to [data structures](/docs/understanding-tracking-design/managing-your-data-structures/ui/index.md) and encapsulate the documented events within a data product.
+
+:::info
+Please note that the creation of event specifications is exclusive to data products.
+:::
+
+Create event specifications in tandem with the latest deployed data structure version in production. This ensures tracking implementation instructions align with the validation criteria in the event specification.
+
+Your event specification guarantees compatibility with the associated data structure version. This ensures events tracked using specified instructions pass validation for the associated data structure.
+
+If a newer event specification version is deployed, consider upgrading event specification and tracking to the latest data structure, ensuring ongoing compliance with validation requirements.
+
+## Creating and Editing Event Specifications
+
+To create a new event specification, follow these steps:
+
+1. Select a data product
+2. Click the "Create event" button
+3. A modal will appear, allowing you to enter the event name
+4. Your first event specification will be displayed on the page
+
+![Create an Event Specification](images/create-event-specification.png)
```

Comment:
These screenshots should be replaced with the updated copy changes I think

Reply:
done